ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -79,7 +79,7 @@ public void Close()
                 emitNamespaceDeclaration = false;
             }
 
-            if (!_config.GenerateMultipleFiles)
+            if (!_config.GenerateMultipleFiles && _outputBuilderFactory.OutputBuilders.Any())
```

Comment:
This avoids us from printing out just a `}` in the case of no `outputBuilders` being present